patroni/ctl.py

```diff
@@ -263,6 +263,8 @@ def get_members(cluster, cluster_name, member_names, role, force, action, schedu
             member_names = role_names
 
     if not member_names and not force:
+        if not candidates:
```

Comment:
At this moment it is already too late:
```
./patronictl.py -c postgres1.yml restart bla
When should the restart take place (e.g. 2019-11-27T17:07)  [now]: 
+---------+--------+------+------+-------+----+-----------+
| Cluster | Member | Host | Role | State | TL | Lag in MB |
+---------+--------+------+------+-------+----+-----------+
+---------+--------+------+------+-------+----+-----------+
Error: bla cluster doesn't have any members
```

Reply:
At least user doesn't need to choose member